controlplane/pkg/tests/nsmd_remote_test.go

```diff
@@ -215,27 +215,27 @@ func TestNSMDDelayRemoteMechanisms(t *testing.T) {
 
 	type Response struct {
 		nsmResponse *connection.Connection
-		err error
+		err         error
```

Comment:
The changes in this file are not relevant to the patch.Please consider a separate PR.